scripts/updateConfigsFromEnvironment.js

```diff
@@ -0,0 +1,68 @@
+/**
+  This script will update the package.json and src/config.json
+  from environment variables.
+
+  Available environment variables:
+    - APP_NAME
+    - APP_VERSION
+    - APP_PUBLISH_URL
+    - APP_BUNDLE_ID
+    - APP_HELP_EMAIL
+    - APP_HELP_SLACK_LINK
+    - APP_ALLOW_PRERELEASE_UPDATES
+ */
+const fs = require('fs')
+const path = require('path')
+const semver = require('semver')
+
+const writeToFile = (relativeFilePath, data) => {
+  const jsonString = JSON.stringify(data, null, 2)
+  const absolutePath = path.join(__dirname, relativeFilePath)
+  fs.writeFile(absolutePath, jsonString, err => {
+    if (err) {
+      console.log(`Error writing file ${absolutePath}`, err)
+    } else {
+      console.log(`Successfully wrote file ${absolutePath}`)
+    }
+  })
+}
+
+const pkg = require('../package.json')
+if (process.env.APP_NAME) {
+  pkg.name = process.env.APP_NAME
+  pkg.build.productName = process.env.APP_NAME
+}
+if (process.env.APP_VERSION) {
+  pkg.version = process.env.APP_VERSION
+}
+if (process.env.APP_PUBLISH_URL) {
+  pkg.build.publish[0].url = process.env.APP_PUBLISH_URL
+}
+if (process.env.APP_BUNDLE_ID) {
+  pkg.build.appId = process.env.APP_BUNDLE_ID
+}
+if (process.env.APP_VERSION_SUFFIX) {
+  const baseVersion = semver(pkg.version, { includePrerelease: true })
```

Comment:
should `includePrerelease` be set to `process.env.APP_ALLOW_PRERELEASE_UPDATES`?

Reply:
Good catch!
This line creates the samver version object. 

My thinking was that if there is a pre-release version in the `pkg.version` it should parse it. But it turns out that this is not really needed since we overwrite the pre-release on the next line

I'm changing this to exclude the `{ includePrerelease: true }`